CDCgov/NEDSS-Modernization
# CNFT2-1174 add static element on page

## Description

An API enabling users to add static elements to their pages

## Tickets

* [CNFT2-1174](https://cdc-nbs.atlassian.net/browse/CNFT2-1174)

## Steps to verify

1. Using swagger, create a static element of each type
2. If you have NBS running locally, you can manually check the database or launch the NBS classic UI and edit the corresponding page to then see whether the static elements created exist


[CNFT2-1174]: https://cdc-nbs.atlassian.net/browse/CNFT2-1174?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
